src/main/resources/init_db.sql

```diff
@@ -1,13 +1,740 @@
+-- Dependencies: 199
+-- Data for Name: users; Type: TABLE DATA; Schema: public; Owner: postgres
+--
+
+COPY public.users (id, login, password, name, surname, token, bucket_id) FROM stdin;
+2	@login	1111	Berry	Garrett	bcecf3f5-06c3-4c23-ae2a-74fdb2649bc8	4
+3	user	2222	Arian	Stark	a1814d6a-e573-4644-95b0-f5f4336e28fe	7
+\.
+
+
+--
+-- TOC entry 2896 (class 0 OID 24758)
+-- Dependencies: 211
+-- Data for Name: users_roles; Type: TABLE DATA; Schema: public; Owner: postgres
+--
+
+COPY public.users_roles (id, user_id, role_id) FROM stdin;
+1	2	1
+2	2	2
+3	3	1
+\.
+
+
+--
+-- TOC entry 2911 (class 0 OID 0)
+-- Dependencies: 204
+-- Name: buckets_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
+--
+
+SELECT pg_catalog.setval('public.buckets_id_seq', 7, true);
+
+
+--
+-- TOC entry 2912 (class 0 OID 0)
+-- Dependencies: 206
+-- Name: buckets_items_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
+--
+
+SELECT pg_catalog.setval('public.buckets_items_id_seq', 26, true);
+
+
+--
+-- TOC entry 2913 (class 0 OID 0)
+-- Dependencies: 196
+-- Name: items_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
+--
+
+SELECT pg_catalog.setval('public.items_id_seq', 10, true);
+
+
+--
+-- TOC entry 2914 (class 0 OID 0)
+-- Dependencies: 200
+-- Name: orders_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
+--
+
+SELECT pg_catalog.setval('public.orders_id_seq', 21, true);
+
+
+--
+-- TOC entry 2915 (class 0 OID 0)
+-- Dependencies: 202
+-- Name: orders_items_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
+--
+
+SELECT pg_catalog.setval('public.orders_items_id_seq', 33, true);
+
+
+--
+-- TOC entry 2916 (class 0 OID 0)
+-- Dependencies: 208
+-- Name: roles_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
+--
+
+SELECT pg_catalog.setval('public.roles_id_seq', 2, true);
+
+
+--
+-- TOC entry 2917 (class 0 OID 0)
+-- Dependencies: 198
+-- Name: users_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
+--
+
+SELECT pg_catalog.setval('public.users_id_seq', 3, true);
+
+
+--
+-- TOC entry 2918 (class 0 OID 0)
+-- Dependencies: 210
+-- Name: users_roles_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
+--
+
+SELECT pg_catalog.setval('public.users_roles_id_seq', 3, true);
+
+
+--
+-- TOC entry 2747 (class 2606 OID 24732)
+-- Name: buckets_items buckets_items_pkey; Type: CONSTRAINT; Schema: public; Owner: postgres
+--
+
+ALTER TABLE ONLY public.buckets_items
+    ADD CONSTRAINT buckets_items_pkey PRIMARY KEY (id);
+
+
+--
+-- TOC entry 2745 (class 2606 OID 24724)
+-- Name: buckets buckets_pkey; Type: CONSTRAINT; Schema: public; Owner: postgres
+--
+
+ALTER TABLE ONLY public.buckets
+    ADD CONSTRAINT buckets_pkey PRIMARY KEY (id);
+
+
+--
+-- TOC entry 2737 (class 2606 OID 24647)
+-- Name: items items_pkey; Type: CONSTRAINT; Schema: public; Owner: postgres
+--
+
+ALTER TABLE ONLY public.items
+    ADD CONSTRAINT items_pkey PRIMARY KEY (id);
+
+
+--
+-- TOC entry 2743 (class 2606 OID 24680)
```

Comment:
@SkaterWeather Are these comments useful?

Reply:
it looks weird

Reply:
yes, it's looks weird, but it's a standart plane-text backup,
On the other side, there is  no sence to modificate auto-generated file